CONTRIBUTING.md

```diff
@@ -1,9 +1,10 @@
 # Instructions for Contributors
 
-As mentioned in the [README](https://github.com/haskell/containers/blob/master/README.md), for reporting bugs (and maybe even the respective fix), please use the [GitHub issue tracker](https://github.com/haskell/containers/issues).
+For reporting bugs (and maybe even the respective fix), please use the [GitHub issue tracker](https://github.com/haskell/containers/issues).
 
 For proposing API changes/enhancements, please follow the [guidelines outlined on the Haskell Wiki](https://wiki.haskell.org/Library_submissions#Guide_to_proposers). Especially note that all API changes/enhancements should be discussed on libraries@haskell.org mailing list.
```

Comment:
"To propose API changes or enhancements"
s/Especially note that all API changes\/enhancements/All such changes

Reply:
Done.